src/config/ai-providers.ts

```
@@ -0,0 +1,223 @@
+      if (!azureConfig.endpoint || !azureConfig.apiKey || !azureConfig.deploymentName) {
+        return null;
+      }
+      
+      return azureConfig;
+
+    case 'openai':
+      const openaiConfig = {
+        provider: 'openai' as const,
+        apiKey: import.meta.env.VITE_OPENAI_API_KEY,
+        baseUrl: import.meta.env.VITE_OPENAI_BASE_URL || 'https://api.openai.com/v1',
+      };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)